plat/common/include/pci/pci_bus.h

```diff
@@ -212,19 +212,19 @@ static struct pci_bus_handler ph __unused;
 
 #define PCI_CONF_VENDOR_ID          (0x00)
 #define PCI_CONF_VENDOR_ID_SHFT     (0)
-#define PCI_CONF_VENDOR_ID_MASK     (0x0000FFFF)
+#define PCI_CONF_VENDOR_ID_MASK     (0xFFFF)
 
 #define PCI_CONF_DEVICE_ID          (0x00)
 #define PCI_CONF_DEVICE_ID_SHFT     (16)
-#define PCI_CONF_DEVICE_ID_MASK     (0x0000FFFF)
```

Comment:
Same comment regarding commit message/subject formatting: too short and not specific. Nevertheless, I don't really see the point of this commit, this is not the only problem with the current PCI subsystem and it should be addressed in the future in the context of a larger overhaul. But ok, I don't have any strong feelings regarding this, keep it if you want.